src/ast/ast.h

```diff
@@ -541,6 +547,16 @@ class While : public Statement
   While(const While &other);
 };
 
+class Scope : public Node
+{
+public:
+  Scope(StatementList *stmts);
+  Scope(const Scope &other) = default;
```

Comment:
Ah, that is true, and I know that - no idea why I put the line there, might have been copied from somewhere and changed because of compiler warnings. Thanks!